### ISSUE TITLE
In Fedora 31, the default fedora source points to Fedora 30

### ISSUE THREAD

**Issue by renich** [osbuild/osbuild-composer#208]
```
# composer-cli sources info fedora 
{"sources":{"Fedora 30":{"name":"Fedora 30","type":"yum-metalink","url":"https://mirrors.fedoraproject.org/metalink?repo=fedora-30\u0026arch=x86_64","check_gpg":true,"check_ssl":true,"system":true}},"errors":[]}
```

**Comment by bcl** [comment]
#210 should fix this

**Comment by major** [comment]
Isn't this fixed by #240 now?
